app/controllers/api/v2/registration_commands_controller.rb

```diff
@@ -32,6 +37,17 @@ def ignored_query_args
       def registration_params
         params[:registration_command]
       end
+
+      def os_with_template?
+        return true unless registration_params[:operatingsystem_id]
+
+        os = Operatingsystem.authorized(:view_operatingsystems).find(registration_params[:operatingsystem_id])
```

Comment:
Don't we have a helper for this? I think it's weird to do this kind of logic on a controller level.